README.md

```diff
@@ -111,3 +111,8 @@ Pull request 歓迎です。以下の条件を満たしていれば、積極的
   - ある条件を満たした一部の社員のみに適用される場合は、フルリモートに含まない
 - 原則として勤務場所が自由
   - 「リモート勤務は週二回まで」などと制限がある場合は、フルリモートに含まない
+
+## How to contribute
```

Comment:
I think this content should be in `Contributing` section, because `Contributing` section has already description about how to add your company.